qic2.py

```diff
@@ -56,6 +56,8 @@
 def doTagging(imageList, startTag, endTag, summary):
     global SITE, debug
     for image in imageList:
+		if 'File:' not in image:
+			image = "File:" + image
         page = pywikibot.Page(SITE, image)
 
         if page.exists():
```

Comment:
This breaks images with `Image:` namespace (totally legal alias), but it doesn’t add the namespace if the namespace occurs anywhere else in the file name (e.g. `This is a File: example.svg`). `pywikibot.FilePage` handles these as expected:
```suggestion
		try:
			page = pywikibot.FilePage(SITE, image)
		except ValueError:
			print("Oops " + image + " isn't a file...")
		else:
			if page.exists():
				# ... and the rest also indented one extra level
```